concent_api/core/models.py

```diff
@@ -518,3 +519,39 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class GlobalTransactionState(Model):
+    """
+    Represents state of whole transaction service by storing last `nonce` used.
+
+    There should always be at most one object of this type with id = 0. If it does not exist, it should be created and
+    nonce initialized with a value obtained from SCI (Client.get_transaction_count()).
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, default=0, unique=True)
+
+    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+        self.pk = 0
+        super().save(*args, **kwargs)
+
+
+class PendingEthereumTransaction(Model):
+    """
+    Represents pending Ethereum transaction state.
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, unique=True)
+
+    gasprice = CharField(max_length=32)
+    startgas = CharField(max_length=32)
```

Comment:
`startgas` in new `SCI` is stored as integer

Reply:
Lets make it `Decimal` then with `max_digits` set to 32.

Reply:
See my comment above: https://github.com/golemfactory/concent/pull/617#discussion_r201396775